lib/providers/notifiers/connection_notifier.dart

```diff
@@ -26,7 +26,6 @@ class ConnectionNotifier extends ChangeNotifier {
     'https://telos.eosphere.io',
     'https://telos.caleos.io',
     'https://api.eos.miami',
-    'https://hyperion.telosgermany.io',
```

Comment:
should we remove this from firebase too. "If we have it there"

Reply:
yah I think it's already been nuked from firebase but I shall check